core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -545,12 +536,14 @@ private BlockStoreLocation commitBlockInternal(long sessionId, long blockId, boo
     BlockStoreLocation loc = tempBlockMeta.getBlockLocation();
 
     // Heavy IO is guarded by block lock but not metadata lock. This may throw IOException.
-    FileUtils.move(srcPath, dstPath);
+    try {
+      FileUtils.move(srcPath, dstPath);
```

Comment:
move actually throws a bunch of runtime exception that should be converted to `AlluxioRuntimeException`.

Reply:
if there's a runtime exception it would be propagated to the caller and get converted anyway